<launch>
  <arg name="width" default="1920" />
  <arg name="height" default="1080" />
  <arg name="fps" default="30" />
  <arg name="frame" default="camera_frame" />
  <arg name="device" default="/dev/video1" />
  <arg name="camera_info_url" default="file://$(find uvc_camera)/example.yaml" />
  <arg name="mode" default="MJPG" />
  <arg name="output_topic" default="bananarama" />
  <node pkg="uvc_camera" type="camera_node" name="uvc_camera" output="screen">
    <param name="width" value="$(arg width)" />
    <param name="height" value="$(arg height)" />
    <param name="fps" value="$(arg fps)" />
    <param name="frame" value="$(arg frame)" />
    <param name="device" value="$(arg device)" />    
    <param name="camera_info_url" value="$(arg camera_info_url)" />
    <param name="output_topic" value="$(arg output_topic)" />
    <param name="mode" value="$(arg mode)" />
  </node>
</launch>
